Lib/test/test_zstd.py

```diff
@@ -2673,8 +2673,12 @@ def test_compress_locking(self):
         input = b'a'* (16*_1K)
         num_threads = 8
 
+        # gh-136394: the first output of comp.compress includes the frame header
+        # so it is different than the others
+        # this is why it is added outside of the threading part
```

Comment:
```suggestion
        # gh-136394: the first output of comp.compress includes the frame header
        # run the first .compress() call outside of the threaded portion to make the test
        # order-independent.
```
Slight suggestion on wording otherwise LGTM!